### PR TITLE
Adding effective wavelength values

### PR DESCRIPTION
**Pull request by mpaillassa**
Following #64, I have added effective wavelength for all surveys.

### CODE REVIEW

**Review by aboucaud — APPROVED** [review]
Great, thanks.

**Comment by mpaillassa** [comment]
Before merging this, I am trying to update the `effective_wavelengths.py` script to plot both the current value and the value computed by speclite for comparison. But when I try to get the current value with `survey.get_filter(filt_name).effective_wavelength` I get that `AttributeError: 'Filter' object has no attribute 'effective_wavelength'` while it works if I try to get other parameters like `exposure_time`. Is it because `effective_wavelength` is marked as `Optional` ?

**Comment by aboucaud** [comment]
Good catch, no it's a bug

**Comment by mpaillassa** [comment]
Hm ok. I was effectively founding that it is [well named ](https://github.com/aboucaud/galcheat/blob/main/galcheat/filter.py#L15)

**Comment by aboucaud** [comment]
Also you should convert it to angstroms before printing so the comparison is easy.

File: scripts/effective_wavelengths.py [hunk 1]
```diff
@@ -25,9 +25,13 @@ def check_effective_wavelengths(survey_name):
...
             speclite_eff_wl = speclite_filt.effective_wavelength
-            # galcheat_eff_wl = survey.get_filter(filt_name).effective_wavelength
+            current_eff_wl = survey.get_filter(filt_name).effective_wavelength
             print(f"  Filter {filt_name} ({speclite_filt_name} in speclite)")
-            print(f"    Speclite effective wavelength: {speclite_eff_wl:.2f}")
+            print(f"    Speclite effective wavelength: {speclite_eff_wl:.3f}")
+            if current_eff_wl is not None:
+                print(f"    Current effective wavelength: {current_eff_wl:.3f}")
+            else:
+                print("    No current value for effective wavelength"
```
Comment (aboucaud) [inline scripts/effective_wavelengths.py:34]:
```suggestion
                print("    No current value for effective wavelength")
```

**Comment by mpaillassa** [comment]
Ok I just converted the speclite value to nm instead to match the unit we use in the yaml files.

Btw we should apply the same print fix for the `zeropoints.py` script. Should we do it here or in another PR ?

**Comment by aboucaud** [comment]
I'll do it in another PR

**Comment by mpaillassa** [comment]
Ok ! 
I also need to find out why `current_eff_wl = survey.get_filter(filt_name).effective_wavelength` is failing on my machine.

**Review by aboucaud — APPROVED** [review]
LGTM

**Pull request merged**